### ISSUE TITLE
Provide a way to initialize a startup Nuecule application 

### ISSUE THREAD

**Issue by tkdchen** [projectatomic/adb-atomic-developer-bundle#181]
When begin to package a container application, "Download a [Nulecule template](https://github.com/projectatomic/nulecule/blob/master/spec/examples/template) to start from." is the first step to get started. It would be much convenient for developers if there is a way to initialize such a directory quickly instead of downloading from the example from Nulecule repo.

For the first step, keep it simple, I would recommend to include `Dockerfile`, `Nulecule`, and `artifacts/`.

**Comment by LalatenduMohanty** [comment]
@tkdchen it is same with https://github.com/projectatomic/adb-atomic-developer-bundle/issues/5 , If you agree we can close this and track it with the old issue.

**Comment by tkdchen** [comment]
@LalatenduMohanty I'm not sure whether I understand #5 correctly, it looks like an issue about documentation.

I was originally intended to provide a tool that can be used by developers directly instead of following a documentation. Let me give two possible scenario,
- a ready-to-use template directory. After I login ADB by `vagrant ssh`, there would be a Nulecule application example directory, then I can `cp -r nulecule-app-template my-nulecule-app` so that I can start package quickly.
- a simple command line utitlity. After I login ADB by `vagrant ssh`, I can execute `generate-nulecule-app` that will generate a read-to-use template directory with all necessary files mentioned above, so that I can start package quickly.

`generate-nulecule-app` is just an example name, it can be anything as long as it's clear enough to know what the utility does.

Both above two items can save me from downloading the template directory from remote Nulecule repo.

If there is something wrong with my understand to #5, please correct me. Thanks. :)

**Comment by bexelbie** [comment]
This isn't quite the same thing as #5, however I think we want to solve this with a combination of #5 and devassistant.  Today, I suggest that if we really need to include these kinds of files we package them up in a container ala Atomic App and then cache that.  However, I think we need to wait on the workflows to get published before we make any decision.

**Comment by tkdchen** [comment]
@bexelbie agree. devassistant would be good to do that. BTW, what does the workflows mean?

**Comment by bexelbie** [comment]
@tkdchen Adi and Ian and Dharmit are working on sample workflows that will show best practice patterns.
